etc/di.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Dotdigitalgroup\Email\Model\Apiconnector\Customer">
+        <plugin name="ddg_enterprise_plugin" type="Dotdigitalgroup\Enterprise\Plugin\CustomerPlugin"
+                sortOrder="1"/>
+    </type>
+    <type name="Dotdigitalgroup\Email\Helper\Data">
+        <plugin name="ddg_helper_plugin" type="Dotdigitalgroup\Enterprise\Plugin\HelperPlugin" sortOrder="1"/>
+    </type>
+    <type name="Dotdigitalgroup\Email\Model\Connector\Datafield">
+        <plugin name="ddg_data_field_plugin" type="Dotdigitalgroup\Enterprise\Plugin\DataFieldPlugin" sortOrder="1"/>
```

Comment:
@adeelq so 3 plugins name that doesn't follow any convention ? : 
can we change to  : 
1. ddg_enterprise_plugin line 4
2. ddg_enterprise_helper_plugin line 8
3. ddg_enterprise_data_field_plugin line 11